### DISCUSSION TITLE
Where to download  data/Vid4/BIx4/calendar/?

### DISCUSSION THREAD

**Discussion by ruanjiyang** [open-mmlab/mmediting#1267] [Datasets] [answered]
I try the BasicVSR by following script:

python demo/restoration_video_demo.py ./configs/restorers/basicvsr/basicvsr_reds4.py https://download.openmmlab.com/mmediting/restorers/basicvsr/basicvsr_reds4_20120409-0e599677.pth data/Vid4/BIx4/calendar/ ./output

but , where to download the data/Vid4/BIx4/calendar/ ?  

I had try to put one video (mp4) file in that directory but it doesn't work. 

Thanks in advance.

**Comment by ckkelvinchan** [comment, marked as answer]
Hi, you can find the Vid4 dataset [here](https://drive.google.com/file/d/1ZuvNNLgR85TV_whJoHM7uVb-XW1y70DW/view?usp=sharing). We will add it to our documentation.

By the way, if you want to do inference on Vid4, you may want to use `https://download.openmmlab.com/mmediting/restorers/basicvsr/basicvsr_vimeo90k_bi_20210409-d2d8f760.pth`

**Comment by ckkelvinchan** [comment]
Documentation has been added. See #399 for more details.
